src/Microsoft.Health.Dicom.SqlServer/Features/Schema/Sql/Sprocs/ISleepIfBusy.sql

```diff
@@ -0,0 +1,111 @@
+﻿/***************************************************************************************/
+-- STORED PROCEDURE
+--    Sleep If Busy
+--
+-- DESCRIPTION
+--    This stored procedure checks if SQL Azure/Server is busy and sleeps if this
+--    is the case. This stored procedure is not very useful on SQL Server, because
+--    service account normally does not have a 'VIEW SERVER STATE' permission.
+--    Note: In on-prem case this sproc does not do anything.
+/***************************************************************************************/
+CREATE OR ALTER PROCEDURE dbo.ISleepIfBusy
```

Comment:
Sproc from Smitha (with some modifications) to inject some waiting between batches